src/main/scala/io/citrine/lolo/trees/Nodes.scala

```diff
@@ -92,7 +92,6 @@ class InternalModelNode[T <: PredictionResult[Any]](
                                                      split: Split,
                                                      left: ModelNode[T],
                                                      right: ModelNode[T],
-                                                     numFeatures: Int,
```

Comment:
`numFeature` parameter is unused here and in `ModelLeaf`

Reply:
The docstring says "used for Shapley computation". I guess that was a lie lol?

Also, if deleting this field, can you update the docstring please?

Reply:
Gregor confirmed it was used in an early iteration but then taken out. I'll update the docstring.